assets/js/sentry.js

```diff
@@ -7,15 +7,16 @@ var loadJS = function(url, implementationCode, location){
 
     location.appendChild(scriptTag);
 };
-var startSentry = function(){
+var startSentry = function() {
     Sentry.onLoad(function() {
         Sentry.init({
-          integrations: [
-            new Sentry.Replay({
-                maskAllText: false,
-                blockAllMedia: false,
-            }),
-          ],
+            integrations: [
+                new Sentry.Replay({
+                    block: ['.table'],
+                    maskAllText: false,
+                    blockAllMedia: false,
+                }),
+            ],
```

Comment:
The spacing changes and other formatting happened automatically by my editor. Let me know if you'd like me to revert those changes.